### PR TITLE
Adding movie inputs

### LINKED ISSUE #24: Adding a Film title to the list once user clicks add button
- creating an event listener that handles a click.
- get the movie title from input.
- creating a div with a class name of movie-name.
- creating an input with class new-film.
- append the bingeWorthy and newFeatureFilm to inputDiv.
- clearing the input title field.
- styling to both classes of (movie-name and new-film).
- changed Hex Codes to CSS Variables.
- clear comments of each line of code within the handler.

### PR DESCRIPTION
**Pull request by PJSalter**
- 🎥  creating an event listener that handles a click.
- 🍿  get the movie title from input.
- 🎟️  creating a div with a class name of movie-name.
- 📼  creating an input with class new-film.
- ✨ append the bingeWorthy and newFeatureFilm to inputDiv.
- 🎞️ clearing the input title field.
- 🎬  styling to both classes of (movie-name and new-film).
- 🎦  changed Hex Codes to CSS Variables.
- 🍬  clear comments of each line of code within the handler.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -54,4 +54,41 @@ function createList() {
...
 
+    // ********************* adding a title for movie in the list *************************** //
+
+    // adding an event listener to this created inputBtn that is inside the beginning binge list box
+inputBtn.addEventListener('click', (e) => {
+  
+  //get the movie title from the input.
+
+  // once the user enters a new movie name, then make a new
+  if(inputTitle.value !== "" && inputTitle.value !== new RegExp("/^\s+$/")) {
```
Comment (mariaalouisaa) [inline index.js:65]: Awesome that you've added this! 🕺 Makes our App seem so PRO!

File: index.js [hunk 2]
```diff
@@ -54,4 +54,41 @@ function createList() {
     const element = document.querySelector('.main')
     element.appendChild(newList)
 
+    // ********************* adding a title for movie in the list *************************** //
+
+    // adding an event listener to this created inputBtn that is inside the beginning binge list box
+inputBtn.addEventListener('click', (e) => {
```
Comment (mariaalouisaa) [inline index.js:60]: I wonder if having the callback function as an arrow function inside the other one may make it harder to test for later...? I think i remember Oli saying it's good to break functions up to smaller functions if possible. Sorry, I don't know if that's super clear (or even correct!)

File: style.css [hunk 3]
```diff
@@ -159,14 +159,14 @@ h1 {
...
+ color: var(--sharp-retroPink);
  position: relative;
  border-radius: 0px;
  font-family:'Gochi Hand', 'Just Another Hand', cursive;
  font-size: 16px;
  border-style: solid;
- border-color: #FF00FF;
- background-color: black;
+ border-color: var(--sharp-retroPink);
+ background-color: var(--onyx-black);
```
Comment (mariaalouisaa) [inline style.css:169]: Thanks for tweaking, loving the consistency 🥳

File: index.js [hunk 4]
```diff
@@ -54,4 +54,41 @@ function createList() {
...
+  if(inputTitle.value !== "" && inputTitle.value !== new RegExp("/^\s+$/")) {
+  e.preventDefault();
+  
+  // create a div with a class of movie-name
+  const bingeWorthy = document.createElement('div');
+  bingeWorthy.classList.add('movie-name');
+  //console.log(bingeWorthy);
+
+  // create an input with class new-film
+  const newFeatureFilm = document.createElement('input');
```
Comment (mariaalouisaa) [inline index.js:74]: I might be being really dim... but what is this new input for?

**Review by mariaalouisaa — COMMENTED** [review]
The code looks great, clear to read I really appreciate the comments and detailed PR message. I just had one question about the new input being created, I'm struggling to imagine / figure out what it's for. Sorry!

File: index.js (hunk 1 above)
Comment (AsmahanM) [reply index.js:65]: Nice use of regex!

File: style.css [hunk 5]
```diff
@@ -180,16 +180,16 @@ h1 {
...
-  color: #FF00FF;
+  color: var(--sharp-retroPink);
   cursor: pointer;
   outline: none;
   width: 15%;
   height: 36px;
   border-radius: 0px;
   border-style: solid;
-  border-color: #FF00FF;
+  border-color: var(--sharp-retroPink);
```
Comment (AsmahanM) [inline style.css:192]: Thanks for switching to the route codes (I noticed a few of mine were still using hex codes).

**Review by AsmahanM — APPROVED** [review]

**Review by mariaalouisaa — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (PJSalter) [reply index.js:65]: Thank you ✨

File: style.css (hunk 3 above)
Comment (PJSalter) [reply style.css:169]: 🤩

File: index.js (hunk 2 above)
Comment (PJSalter) [reply index.js:60]: This was a way that I am used to writing event listeners with the arrow function, and a challenge that I got this handler to work was by placing this inside the other, if this causes issues then I can modify this once we create the tests, these are all valid point's, thank you for the feedback ✨

File: index.js (hunk 4 above)
Comment (PJSalter) [reply index.js:74]: It's an input within another input, think about more stretch goals later then we can add an edit functionality later on ⭐

Comment (PJSalter) [reply index.js:74]: *I was thinking of more stretch goals sorry for typo 😆

File: style.css (hunk 5 above)
Comment (PJSalter) [reply style.css:192]: your very welcome 😸

**Pull request merged**